ui/packages/components/src/Forms/Input.tsx

```diff
@@ -14,7 +15,20 @@ const sizeStyles = {
 };
 
 export const Input = forwardRef<HTMLInputElement, InputProps>(
-  ({ className, type = 'text', inngestSize = 'base', ...props }, ref) => {
+  (
+    { allowPasswordManager = false, className, type = 'text', inngestSize = 'base', ...props },
+    ref
+  ) => {
+    let passwordManagerProps: Record<string, unknown> = {
+      autocomplete: 'off',
+      'data-1p-ignore': true,
+      'data-bwignore': true,
+      'data-lpignore': true,
```

Comment:
I didn't realize, but since this is react, do these props need to be camelCase? e.g. autocomplete is really `autoComplete` in react: https://legacy.reactjs.org/docs/dom-elements.html#all-supported-html-attributes

Reply:
For whatever reason I usually see `data-` attributes written in kebab-case. Dunno why

Reply:
It seems from testing, React doesn't care when passing props like this, even for `autoComplete`. Not sure if there is a warning in the console, but I'm good with shipping this then.